lib/src/component_declaration/annotations.dart

```diff
@@ -256,20 +331,6 @@ class Accessor {
   });
 }
 
-/// Deprecated.
-///
-/// Use [Accessor], [requiredProp], or [nullableRequiredProp] instead.
-@Deprecated('2.0.0')
-class Required {
```

Comment:
are we allowed to remove this in a minor because it was marked as deprecated 2 versions ago? or is it bad since it wasn't remove in the 3.0.0 major?

Reply:
Shhhhhh